rpcserver.go

```diff
@@ -2523,15 +2554,41 @@ func (r *rpcServer) AssetRoots(ctx context.Context,
 	return resp, nil
 }
 
+func UnmarshalUniProofType(rpcType unirpc.ProofType) (universe.ProofType,
```

Comment:
nit: missing Godoc. Also, should we move these marshaling functions into `taprpc/marshal.go`?

Reply:
Can't move into `taprpc/marshal.go` because of `universe` package import.